### ISSUE TITLE
List of texts replaced with same replacement

### ISSUE THREAD

**Issue by steenstn** [Woundorf/foxreplace#173]
Great plugin! Only thing I am missing is the option to have a list of texts replaced with the same replacement for texts that are not easily regex-matched. So a many-to-one relationship instead of many one-to-one relationships.

So I can say e.g.
- Banana
- Apple
- Pear
all gets replaced by "Fruit" instead of defining
- Banana is replaced by Fruit
- Apple is replaced by Fruit
- Pear is replaced by Fruit

**Comment by Woundorf** [comment]
These words can actually be matched with a regex: __banana|apple|pear__

**Comment by steenstn** [comment]
That is true, but I have multiple sentences that I would like to replace with one sentence and while possible with regex it would be a damn long one :)

**Comment by Woundorf** [comment]
Ok, then I will leave the issue open but with low priority because I prefer to prioritize things that are broken or that can't be done in any way.

**Comment by steenstn** [comment]
Fair enough. Your comment gave me an idea so I got it to suit my needs with a ^Sentence1Start.*|^Sentence2Start.* etc. so I'm happy anyway!
